editor/scene_tree_dock.cpp

```diff
@@ -1963,6 +1967,54 @@ bool SceneTreeDock::_update_node_path(Node *p_root_node, NodePath &r_node_path,
 	return false;
 }
 
+_ALWAYS_INLINE_ bool recurse_into_property(const PropertyInfo &p_property) {
+	// Only check these types for NodePaths.
+	static const Variant::Type property_type_check[] = { Variant::OBJECT, Variant::NODE_PATH, Variant::ARRAY, Variant::DICTIONARY, Variant::RID };
```

Comment:
What's RID for?

Reply:
I don't remember why I put that there. I think it can go, there rest of the code does nothing with RIDs.